### PR TITLE
Display routes nyanhorse

### PR DESCRIPTION
**Pull request by NyanHorse**
- searchView functionallity working
- new fetch call for the routes
- new fetch call for the places
- new api call for the routes
- new api call for the locations
- locationMap added (Mapbox)
- routeView added

### CODE REVIEW

File: src/components/searchView.jsx [hunk 1]
```diff
@@ -1,15 +1,99 @@
...
+    for (let key in routes) {
+      routesArr.push(
+        <div className="route" key={key} onClick={() => this.showPlace(key, routes[key].id)}>
+          <p>{routes[key].name}</p>
+          <p>{routes[key].description}</p>
+        </div>);
+    }
+
+    if (this.state.loading) {
+      showed = <div></div>;
```
Comment (kmoroder) [inline src/components/searchView.jsx:67]: Maybe add a spinner here?

File: src/components/searchView.jsx [hunk 2]
```diff
@@ -1,15 +1,99 @@
...
+    } else {
+      showed = routesArr;
```
Comment (kmoroder) [inline src/components/searchView.jsx:69]: I'm not sure I like the name "showed" - can you find something a little more descriptive? Even "routes" would be better.

**Review by kmoroder — APPROVED** [review]
A few comments, but no need to change in this pull request.

File: src/components/searchView.jsx (hunk 1 above)
Comment (NyanHorse) [reply src/components/searchView.jsx:67]: future feature

File: src/components/searchView.jsx (hunk 2 above)
Comment (NyanHorse) [reply src/components/searchView.jsx:69]: sure!

**Pull request merged**